Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
I also get this if I `git pull` inside a `root:wheel` brew directory:

```
error: unable to unlink old '.gitignore' (Permission denied)
error: unable to unlink old '.travis.yml' (Permission denied)
error: unable to unlink old '.yardopts' (Permission denied)
error: unable to unlink old 'CODEOFCONDUCT.md' (Permission denied)
error: unable to unlink old 'CONTRIBUTING.md' (Permission denied)
error: unable to unlink old 'LICENSE.txt' (Permission denied)
error: unable to unlink old 'README.md' (Permission denied)
error: unable to unlink old 'SUPPORTERS.md' (Permission denied)
fatal: Could not reset index file to revision 'origin/master'.
```

Everything inside that directory is `Dominyk:admin` but the directory itself is `root:wheel` as per what happens to many people on El Cap upgrade. I'm just wary on the above point that `brew repair-permissions` wouldn't be able to use `sudo` to repair the `/usr/local` permissions and sometimes repairing the permissions inside the directory isn't enough.